noteApp/note.js

```diff
@@ -1,7 +1,17 @@
 function Note() {
-
+  this.noteStore = [];
 }
 
 Note.prototype.helloWorld = function () {
   return 'Hello World!'
 };
+
+Note.prototype.create = function (newNote) {
+  this.noteStore.push(newNote);
+};
+
+Note.prototype.viewNote = function () {
```

Comment:
If this method will be used to view all notes eventually, we should call it `viewNotes`

Reply:
Change made